main.tf

```diff
@@ -213,7 +213,7 @@ exec > >(tee /var/log/user-data.log|logger -t user-data ) 2>&1
 echo "#Pull values from Parameter Store and save to profile"
 touch /home/tableau_srv/env_vars.sh
 echo "
-export S3_HAPROXY_CONFIG_BUCKET=${var.s3_haproxy_config_bucket}
+export S3_HAPROXY_CONFIG_BUCKET=${var.s3_httpd_config_bucket}
```

Comment:
To be in line with all changes the variable name may also need changing from `S3_HAPROXY_CONFIG_BUCKET` to `S3_HTTPD_CONFIG_BUCKET`

Reply:
Corrected in latest commit.